src/entries/background.ts

```diff
@@ -9,6 +9,8 @@ export default defineBackground({
     const storage = new Storage({
       area: "local"
     })
+
+    let isSidePanelOpen: boolean = false
```

Comment:
The wording should be updated as well. "Open Side Panel" -> "Close Side Panel" or just "Toggle Side Panel"